metagraph/src/annotation/binary_matrix/row_sparse/row_sparse.hpp

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <fstream>
+#include <string>
+#include <vector>
+
+#include <sdsl/enc_vector.hpp>
+
+#include "annotation/binary_matrix/base/binary_matrix.hpp"
+#include "common/vectors/bit_vector_adaptive.hpp"
+
+namespace mtg {
+namespace annot {
+namespace binmat {
+
+class RowSparse : public BinaryMatrix {
+  public:
+    RowSparse() {}
+    RowSparse(const std::function<void(const RowCallback&)> &call_rows,
+                    uint64_t num_columns,
+                    uint64_t num_rows,
+                    uint64_t num_relations);
+    RowSparse(RowSparse &&);
```

Comment:
Looks like everything is default.

Maybe just
```suggestion
    RowSparse(RowSparse &&) = default;
```